.github/workflows/nvqc_regression_tests.yml

```diff
@@ -0,0 +1,351 @@
+name: Nvqc regression tests
+
+concurrency:
+  # only one integration tests workflow to be run at a time, since it involves pushing/cleaning up a test image (same tag).
+  group: ${{ github.workflow }}${{ github.event.workflow_run.name }}
+  cancel-in-progress: false
+
+# Run on request and every day at 3 AM UTC
+on:
+  workflow_dispatch:
+    inputs:
+      target:
+        description: 'Target (choose nightly to run like nightly tests)'
+        required: true
+        default: 'nightly'
```

Comment:
I'm wondering if this `target` input parameter applies to this new test. We might be able to do without it.